graph/fetch-schema.js

```diff
@@ -0,0 +1,17 @@
+import fetch from 'node-fetch';
+import introspectionQuery from './introspection-query';
+
+const baseGraphUrl = 'https://graphql.myshopify.com/api/graph';
+
+export default function fetchSchema() {
+  return fetch(baseGraphUrl, {
+    method: 'POST',
+    headers: {
+      Authorization: 'Basic MzUxYzEyMjAxN2QwZjJhOTU3ZDMyYWU3MjhhZDc0OWM=',
```

Comment:
It's not a secret, so I'm not tooooo worried about it. Ideally I'd like to have a common config that devs can use where they can pull the schema from their own shop. Until then: recommendation for where this should go? Or OK here? (this branch is prototype-y)